web/client/plugins/widgets/WidgetsTray.jsx

```diff
@@ -23,7 +23,7 @@ import BButton from '../../components/misc/Button';
 import {mapLayoutValuesSelector} from "../../selectors/maplayout";
 
 const Button = tooltip(BButton);
-
+const singleWidgetLayoutBreakpoint = 600;
```

Comment:
provide this breakpoint as a prop with default value 600.

Reply:
provided via the patch you've posted, thanks